app.js

```diff
@@ -19,7 +19,28 @@ app.get('/', (req, res) => {
   res.render('Home');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      console.log('All the Beers from the API: ', beersFromApi);
+      const data = { beersFromApi: beersFromApi };
+      res.render('Beers', data);
+    })
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beer', (req, res, next) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => {
+      const data = { responseFromAPI: responseFromAPI };
```

Comment:
In this line you can just say:

const data = { responseFromAPI: responseFromApi[0] }

So you don't have to repeat the [0] everytime you call it in the RandomBeer file